project/MimaExcludes.scala

```diff
@@ -40,7 +40,22 @@ object MimaExcludes {
     ProblemFilters.exclude[MissingClassProblem]("org.apache.spark.sql.connector.write.V1WriteBuilder"),
 
     // [SPARK-33955] Add latest offsets to source progress
-    ProblemFilters.exclude[DirectMissingMethodProblem]("org.apache.spark.sql.streaming.SourceProgress.this")
+    ProblemFilters.exclude[DirectMissingMethodProblem]("org.apache.spark.sql.streaming.SourceProgress.this"),
+
+    // [SPARK-34862][SQL] Support nested column in ORC vectorized reader
```

Comment:
This is weird, where do we change `org.apache.spark.sql.vectorized.ColumnVector` in this PR?

Reply:
@cloud-fan - yeah it's weird. We don't change `ColumnVector` class at all. Do you have any idea for how to debug on this? I am still checking why, thanks.

Reply:
maybe it's some bugs in Mima, not a bit deal as we know this PR doesn't break binary compatibility.